src/common/ZulipStatusBar.js

```diff
@@ -55,27 +54,26 @@ class ZulipStatusBar extends PureComponent<Props> {
   };
 
   render() {
-    const { theme, hidden, orientation } = this.props;
+    StatusBar.setTranslucent(true);
```

Comment:
Interesting; what's the reason this is included in the same commit vs. a different commit? They might go together; it just hasn't been made obvious to me.

Also, what's the reason for using `.setTranslucent` instead of using the [`translucent` prop](https://reactnative.dev/docs/statusbar#translucent-android)?

Reply:
Its my fault, I just saw and used it, both caters the same purpose and it would be better to use it as prop.
And about
>same commit vs. a different commit?

Its been around 18 days, as I remember I couldn't found a commit message to describe changes that i removed a condition that restrict statusbar to be visible in landscape mode in android but it was not catering the purpose and in landscape mode default status-bar had been visible.
For the next, `translucent` will make lightbox visible in full screen without jumping, [CZO](https://chat.zulip.org/#narrow/stream/48-mobile/topic/status-bar/near/1164588)
I will do it :) .

I will be very thankfull to you, If you test it on IOS. I don't have macOS

Reply:
mentioning you @chrisbobbe  :) would be a good way to notify, thank you